src/node_dependencies/index.js

```diff
@@ -35,20 +35,27 @@ const isNotJunk = function (file) {
 }
 
 // Retrieve all the files recursively required by a Node.js file
-const getDependencies = async function (mainFile, srcDir, pluginsModulesPath) {
+const getDependencies = async function (mainFile, srcDir, pluginsModulesPath, moduleAllowList) {
```

Comment:
We want to get a list of all the files to be included for a given dependency. I've repurposed this method and added a `moduleAllowList` parameter — when defined, only the files belonging to these dependencies will be returned.